requirements-common.txt

```diff
@@ -32,3 +32,5 @@ six>=1.16.0; python_version > '3.11' # transitive dependency of pandas that need
 setuptools>=74.1.1; python_version > '3.11' # Setuptools is used by triton, we need to ensure a modern version is installed for 3.12+ so that it does not try to import distutils, which was removed in 3.12
 einops # Required for Qwen2-VL.
 compressed-tensors == 0.8.0 # required for compressed-tensors
+runai-model-streamer
+boto3
```

Comment:
I'm not sure if these make sense as required dependencies iirc boto3 is quite large

Reply:
Indeed installing boto3 and all its deps is large (47MB)

Another option for implementation is to read the s3 metadata files using the Run:ai model streamer instead of boto3
The Run:ai Model Streamer has lots of common dependencies with vLLM so installing it is just a 1MB addition.

Anyway, I have made it optional dependency (vllm[runai] -> boto3 + runai-model-streamer + [s3])
And I also added it to the OpenAI API Dockerfile so people can use s3 capabilities out of the box (The image is more than 10GB so 47MB relatively small)